custom/enikshay/integrations/ninetyninedots/repeater_generators.py

```diff
@@ -263,9 +263,12 @@ class AdherencePayloadGenerator(NinetyNineDotsBasePayloadGenerator):
 
     def get_payload(self, repeat_record, adherence_case):
         domain = adherence_case.domain
+        episode_case = get_episode_case_from_adherence(domain, adherence_case.case_id)
+        if episode_case.closed:
```

Comment:
This seems awkward but mimics the current behavior of ```get_payload```.  Let me know if you have suggestions. @proteusvacuum